src/platform_impl/linux/drm/event_loop.rs

```diff
@@ -0,0 +1,1417 @@
+                                    }
+
+                                | xkb_keymap::XKB_KEY_Meta_L // LMeta
+                                    | xkb_keymap::XKB_KEY_Meta_R // RMeta
+                                    => {
+                                        match state {
+                                            ElementState::Pressed => self.modifiers |= ModifiersState::LOGO,
+                                            ElementState::Released => self.modifiers.remove(ModifiersState::LOGO)
+                                        }
+                                        callback(crate::event::Event::WindowEvent {
+                                            window_id: crate::window::WindowId(crate::platform_impl::WindowId::Drm(super::WindowId)),
+                                            event: crate::event::WindowEvent::ModifiersChanged(self.modifiers)}, &mut ());
+                                    }
+                                xkb_keymap::XKB_KEY_Sys_Req | xkb_keymap::XKB_KEY_Print
+                                    => {
+                                        if self.modifiers.is_empty() {
+                                            callback(crate::event::Event::WindowEvent {
+                                                window_id: crate::window::WindowId(crate::platform_impl::WindowId::Drm(super::WindowId)),
+                                                event: crate::event::WindowEvent::CloseRequested
+                                            }, &mut ());
+                                        }
+                                    }
+                                _ => {}
+                            }
+                    }
+                    _ => {}
+                }
+            }
+        }
+        Ok(PostAction::Continue)
+    }
+
+    fn register(&mut self, poll: &mut Poll, factory: &mut TokenFactory) -> std::io::Result<()> {
+        self.token = factory.token();
+        poll.register(self.as_raw_fd(), Interest::READ, Mode::Level, self.token)
+    }
+
+    fn reregister(&mut self, poll: &mut Poll, factory: &mut TokenFactory) -> std::io::Result<()> {
+        self.token = factory.token();
+        poll.reregister(self.as_raw_fd(), Interest::READ, Mode::Level, self.token)
+    }
+
+    fn unregister(&mut self, poll: &mut Poll) -> std::io::Result<()> {
+        self.token = Token::invalid();
+        poll.unregister(self.as_raw_fd())
+    }
+}
+
+/// An event loop's sink to deliver events from the Wayland event callbacks
+/// to the winit's user.
+type EventSink = Vec<crate::event::Event<'static, ()>>;
+
+pub struct EventLoopWindowTarget<T> {
+    /// drm Connector
+    pub connector: drm::control::connector::Info,
+
+    /// drm crtc
+    pub crtc: drm::control::crtc::Info,
+
+    /// drm mode
+    pub mode: drm::control::Mode,
+
+    /// drm plane
+    pub plane: drm::control::plane::Handle,
```

Comment:
All doc comments must start with capital character. So `Drm connector.`, `Drm crtc` and so on.

Reply:
Fixed